src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -995,6 +1004,11 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         {
             if (point.Properties().IsLeftButtonPressed())
             {
+                if (_clickDragStartPos)

```

Comment:
Could you add a comment here about what this accomplishes? Just basically explain what bug it fixes.